packages/dashboard/src/CdcDashboardComponent.tsx

```diff
@@ -656,7 +694,7 @@ export default function CdcDashboard({ initialState, isEditor = false, isDebug =
                       dataConfig={config.datasets[datasetKey]}
                       rawData={config.datasets[datasetKey].data?.[0]?.tableData || config.datasets[datasetKey].data}
                       runtimeData={config.datasets[datasetKey].data?.[0]?.tableData || filteredTableData || config.datasets[datasetKey].data || []}
-                      expandDataTable={config.table.expanded}
```

Comment:
I think this should probably stay as `config.table.expanded`. When I check expand data table in the dashboard settings the data table doesn't expand by default anymore.